client/src/components/UserSidebar/UserSidebar.jsx

```diff
@@ -29,12 +39,16 @@ const UserSidebar = (props) => {
           subtitle={user ? user.email : ''}
           title={user ? user.username : ''}
         />
+        <CardActions>
```

Comment:
On second thought, I think it'll be better to handle this when you're implementing profile views for other users.